src/webots/gui/WbGuiApplication.cpp

```diff
@@ -305,7 +306,7 @@ void WbGuiApplication::parseArguments() {
       args.pop_front();
       const QStringList streamArgument = parser.values(streamOption);
       if (streamArgument.size() != 1) {
-        cerr << tr("webots: invalid option: '--log-performance': log file path is missing.").toUtf8().constData();
+        cerr << tr("webots: invalid option : '--log-performance': log file path is missing.").toUtf8().constData();
```

Comment:
In English, there is no space before a colon:
```suggestion
        cerr << tr("webots: invalid option: '--log-performance': log file path is missing.").toUtf8().constData();
```